test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -501,8 +497,8 @@ def __init__(self, out_dir, diag_path=None, server_host=None,
         self.mapfile = map_file
 
         # Initialize Server and Scheduler objects
-        self.server = Server(server_host, diag=self.diag)
-        self.scheduler = Scheduler(server=self.server, diag=self.diag)
+        self.server = Server(server_host)
+        self.scheduler = Scheduler(server=self.server)
```

Comment:
You're no longer calling this with self.diag, but I didn't see self.diag get removed.  Is it still being set or initialized somewhere?